### ISSUE TITLE
Non-existing unrecognized file cannot be deleted

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dizzydezz/jmm#368]
```
What steps will reproduce the problem?
1. Add a file which won't be found on AniDB
2. Run the JMM import -> the file is added to the 'unrecognized' list
3. Delete the file from disk
4. Notice that the file can't be removed from the list: Delete returns a 'file 
not found' error, and upon rescanning the file remains in the list.

What version of the product are you using? On what operating system?
Lastest version as of December 30, 2012.
No idea which version exactly though - where can you find the version number?

Please provide any additional information below.
Workaround: create a dummy file with the same name, thenclick 'delete' in the 
unrecognized file list.


```

Original issue reported on code.google.com by `bert.rey...@gmail.com` on 30 Dec 2012 at 3:04

**Comment by GoogleCodeExporter** [comment]
Original comment by `werndly...@gmail.com` on 30 Dec 2012 at 11:36
- Changed state: **Accepted**
- Added labels: **Priority-High**, **Product-JMM**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 3.1.28
```

Original comment by `werndly...@gmail.com` on 3 Jan 2013 at 1:19
- Changed state: **Fixed**
